Readme.md

```diff
@@ -3,6 +3,8 @@
 
  X-Response-Time middleware for koa.
 
+___Notice: `koa-response-time@2` supports `koa@2`; if you want to use this module with `koa@1`, please use `koa-response-time@1`.___
```

Comment:
we should be using peerDependencies for this sort of thing I think npm resolves it correctly automatically